tools/common.props

```diff
@@ -14,7 +14,4 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Key.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Net.Compilers" Version="$(RoslynVersion)" PrivateAssets="All" />
```

Comment:
Opened issue #1536. We were not using the right compiler to begin with (it should be Microsoft.NetCore.Compilers). That package was built against framework 1.1.0 and hence using it on a machine that doesn't have 1.1.0 and only has the dotnet cli that we bring down does not work.